experiments/functional/jiva-volume-resize/test.yml

```diff
@@ -0,0 +1,328 @@
+  connection: local
+
+  vars_files:
+    - test_vars.yml
+
+  tasks:
+    - block:
+
+          ## Generating the testname for deployment
+        - include_tasks: /utils/fcm/create_testname.yml
+
+          ## RECORD START-OF-TEST IN LITMUS RESULT CR
+        - include_tasks: /utils/fcm/update_litmus_result_resource.yml
+          vars:
+            status: 'SOT'
+        
+        - name: Update the DaemonSet template with the variables.
+          template:
+            src: daemonset.j2
+            dest: daemonset.yml
+
+        - name: Create the DaemonSet 
+          shell: kubectl apply -f daemonset.yml
+          args:
+            executable: /bin/bash
+          register: status
+          failed_when: "status.rc != 0"
+
+        - name: Obtain the application pod
+          shell: >
+            kubectl get po -n {{ app_ns }} -l {{ app_label }} 
+            --no-headers -o custom-columns=:.metadata.name
+          args:
+            executable: /bin/bash
+          register: app_pod
+          failed_when: "app_pod.rc != 0"
+
+        - name: Record the application pod name
+          set_fact: 
+            application_pod: "{{ app_pod.stdout }}"
+
+        - name: Verify if the application pod is running
+          shell: >
+            kubectl get po -n {{ app_ns }} -l {{ app_label }} 
+            --no-headers -o custom-columns=:.status.phase
+          args:
+            executable: /bin/bash
+          register: app_status
+          until: "'Running' in app_status.stdout"
+          delay: 15
+          retries: 60
+
+        - name: Obtain the node where application pod is scheduled
+          shell: >
+            kubectl get po "{{ app_pod.stdout }}" -n {{ app_ns }} 
+            --no-headers -o custom-columns=:.spec.nodeName
+          args:
+            executable: /bin/bash
+          register: app_node
+          failed_when: "app_node.rc != 0"
+
+        - name: Obtain the node IP where application pod is scheduled
+          shell: >
+            kubectl get node "{{ app_node.stdout }}" 
+            -o jsonpath='{.status.addresses[0].address}'
+          args:
+            executable: /bin/bash
+          register: node_ip
+          failed_when: "node_ip.rc != 0"
+
+        - name: Obtain the IP address of jiva controller pod
+          shell: >
+            kubectl get pod -n {{ app_ns }} -l openebs.io/controller=jiva-controller 
+            --no-headers -o custom-columns=:.status.podIP
+          args:
+            executable: /bin/bash
+          register: controller_ip
+          failed_when: "controller_ip.rc != 0"
+                  
+        - name: Getting PV name from the pvc
+          shell: >
+            kubectl get pvc {{ app_pvc }} -n {{ app_ns }} 
+            --no-headers -o custom-columns=:.spec.volumeName
+          args:
+            executable: /bin/bash
+          register: app_pv
+          failed_when: "app_pv.rc != 0"
+
+        - name: Obtain the ds pod corresponding to application pod
```

Comment:
don't use alias in task name. ds - daemonset

Reply:
resolved